### DISCUSSION TITLE
Possibility to include winddelen.nl

### DISCUSSION THREAD

**Discussion by Darkguver** [jobvk/Home-Assistant-Windcentrale#95] [Ideas]
Hi,

i was reading https://github.com/jobvk/Home-Assistant-Windcentrale/discussions/69.
and i was wondering if it's possible to incorporate the mijn.winddelen.nl in this plugin 
since both are using nearly the same backend.

Or is this plugin soly for the windcentrale?

**Comment by jobvk** [comment]
Would be difficult because it uses a different AWS server for authentication.  So I have no login for testing and developing.
So I think including is not a option. But i'm open to develop a fork of this repo with you for winddelen.nl

**Reply by Darkguver** [reply]
i have the implementation working with minimal changes (they are using the same backend and structure)
the url changes from: in wind.py `mijn.windcentrale.nl` to `mijn.winddelen.nl`
and the AWS changes to `pool_id='eu-west-1_3ujjjPxxH', client_id='11edho7vncqa74o1bju4tlgnt0'`
everythin else is the same

**Reply by arndtje** [reply]
Works like a charm!

**Reply by jobvk** [reply]
@Darkguver @arndtje

I made a beta release [0.2.7-Beta](https://github.com/jobvk/Home-Assistant-Windcentrale/releases/tag/0.2.7-Beta) that supports mijn.winddelen.nl.

Could you guys please check if it works and give me some feedback?

You can install it by HACS.
You have to remove the component and add the again, then it should work.

**Reply by Darkguver** [reply]
I have updated the integration, and it looks like it works like a charm

**Comment by jobvk** [comment]
I made a beta release [0.2.7](https://github.com/jobvk/Home-Assistant-Windcentrale/releases/tag/0.2.7) that supports mijn.winddelen.nl.
